docs/reference/2-protocols/2-device-auth.md

```diff
@@ -3,20 +3,67 @@ id: device-auth
 title: Device Authentication
 ---
 
-To access Golioth Service from a device, the device need to be properly created and provisioned.
+To access Golioth Services from a device, the device needs to be properly created and provisioned.
 
-You can manage device and project credentials using the Web Console or through the CLI.
+You can manage device level and project credentials using the Web Console or through the CLI.
 
-### Credential Types
+## CoAP Gateway address and port
 
-| Credential Type               | Description               | Supported Gateways    |
-| ----------------------------- | ------------------------- | --------------------- |
-| Per Device PSK/Pre Shared Key | Pre Shared Keys           | CoAP ✅               |
-| Per Project Certificate       | Mutual TLS Authentication | CoAP 🚧               |
+Devices can connect to the Golioth CoAP Gateway at the following endpoint:
 
-### Creating an Pre Shared Key Credential
+ - `coap.golioth.io` at UDP port `5684`, using DTLS
 
-See the getting started guide to [authorize devices](/getting-started/3-commandline/6-authorize-devices.md).
+Devices that use URL can connect using `coaps://coap.golioth.io/`.
 
-- Guides on how to use:
-  - [Using Pre Shared Keys on CoAP Gateway](/reference/protocols/coap/auth)
+The CoAP gateway requires authentication.
+
+## Authentication Methods
+
+| Authentication Method         | Description                             |
+| ----------------------------- | --------------------------------------- |
+| DTLS PSK                      | Per-device pre-shared key               |
+| DTLS Client Certificate       | Mutual certificate-based authentication |
```

Comment:
Minor: This one is also "per-device". If you leave that phrase out, it might imply to some readers that it's not per-device.

Reply:
This is hard to formulate in one short description, because the action that a user needs to perform on Golioth is not per device. So I'd rather have some people thinking it's not per device, rather than have people think it's per device while the actual feature is about not having to register certificate of every device.